bazel/webassembly.bzl

```diff
@@ -0,0 +1,102 @@
+"""
```

Comment:
This file was adapted from `bazel/canisters.bzl`.

If the rust_wasm_bindget(_test) works, then this file is non longer needed.